cms/templates/widgets/footer.html

```diff
@@ -0,0 +1,36 @@
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+%>
+
+<div class="wrapper-footer wrapper">
+  <footer class="primary" role="contentinfo">
+
+    <div class="footer-content-primary">
+      <div class="colophon">
+        <p>&copy; ${settings.COPYRIGHT_YEAR} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
+        <p><a data-rel="edx.org" href="https://lms.mitx.mit.edu/tos" target="_blank">${_("Terms of Service")}</a></p>
```

Comment:
I just wanted to double-check that the URL added by your PR https://github.com/mitodl/mitx-theme/pull/13 was `/tos`